.travis.yml

```diff
@@ -27,6 +27,8 @@ env:
   - TEST_TARGET="test-suite" COMPILER="4.02.3+32bit"
   - TEST_TARGET="validate"                           TW="travis_wait"
   - TEST_TARGET="validate"   COMPILER="4.02.3+32bit" TW="travis_wait"
+  - TEST_TARGET="ci-bedrock-src"                     TW="travis_wait 45"
```

Comment:
Ugh, but `travis_wait` doesn't display the output as the target builds.  Can you fix this from your side?  Possibly with something like https://github.com/JasonGross/bedrock/blob/master/.travis.yml#L46

In any case, I guess I'll need to track down the performance changes between 8.4 and 8.6, because it seems like coq+bedrock is a bit too slow for travis.